sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ClientEncryptionPolicy.java

```diff
@@ -40,9 +40,25 @@ public final class ClientEncryptionPolicy {
      */
     @Beta(value = Beta.SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public ClientEncryptionPolicy(List<ClientEncryptionIncludedPath> paths) {
-        this.validateIncludedPaths(paths);
-        this.includedPaths = paths;
         this.policyFormatVersion = 1;
+        this.validateIncludedPaths(paths, policyFormatVersion);
+        this.includedPaths = paths;
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param paths list of path of the item that need encryption along with path-specific settings.
+     * @param policyFormatVersion version of the client encryption policy definition. Current supported versions are 1 and 2. Default version is 1.
```

Comment:
If the current encryption package will start failing with a new minor bump of the underlying SDK (because SDK creates v2 policies, and encryption package only understands v1), we should avoid changing the default and let the customer opt-in - we did the same on the .net implementation.

Reply:
Since we are planning to do a major version bump, that's why I am wondering if it makes more sense to change it to v2, if that's the future route for encryption. At least with a major version, we can justify the breaking change. 
Otherwise, in future, it will always be an opt-in.

Reply:
I expect the SDK version bump itself will be minor. Unless we somehow prevent using this bumped SDK with encryption v1.x package, a customer who just bumps the SDK version will see encryption not work on a container that they create using this bumped SDK version.

Reply:
+1 on Abhijit's comment

Reply:
Updated the version to 2.0.0-beta.1